services/NotificationAgent/src/notification_agent/common.clj

```diff
@@ -17,33 +15,18 @@
       (cheshire/decode-stream (reader body) true)
       (cheshire/decode body true))
     (catch Throwable t
-      (throw+ {:error_code ce/ERR_INVALID_JSON
-               :defails    (.getMessage t)}))))
+      (throw+ {:type    :clojure-commons.exception/invalid-json
+               :defails (.getMessage t)}))))
```

Comment:
Heh, defails. I'm guessing that's supposed to be a t, not that it's in your changes :)